packages/experimental/Shimmer/src/Shimmer.tsx

```diff
@@ -0,0 +1,99 @@
+/** @jsx withSlots */
+import { useState, useEffect, useMemo } from "react";
+import { Circle, ClipPath, Defs, LinearGradient, Rect, Stop, Svg, Image } from 'react-native-svg';
+import { shimmerName, ShimmerProps, ShimmerType } from './Shimmer.types';
+import { mergeProps } from '../../../framework/merge-props/lib/mergeProps';
+import { buildUseStyling, compose, UseSlots, withSlots } from '../../../experimental/framework/lib';
```

Comment:
nit: importing from relative path instead of "@fluentui-react-native/framework" or whatever the package name is